cogs/AI.py

```diff
@@ -13,7 +13,10 @@ def __init__(self, bot: OGIROID):
         self.bot = bot
 
     @commands.slash_command(description="Generates ai art")
-    async def ai_art(self, inter: disnake.ApplicationCommandInteraction, text):
+    async def ai_art(self, inter: disnake.ApplicationCommandInteraction, text: str):
+        if profanity.censor(text) != text:
+            return await inter.send(f"NSFW requests are not allowed!",
+                                    ephemeral=True)
```

Comment:
Interesting choice. Instead of this, you can also censor parts if it's a long command?

Reply:
what do you mean this will just stop the command if any bad words are used it doesn't change prompt

Reply:
> what do you mean this will just stop the command if any bad words are used it doesn't change prompt

`profanity.censor(text)` This part censors the bad words. So you can just use that as the prompt maybe? If it's a really long prompt and there's only one word then it makes sense to just censor that one. But if the whole prompt is one word, then stop the command